### DISCUSSION TITLE
npm- Unable to Generate xlsx file (and csv). Able to generate html, json

### DISCUSSION THREAD

**Discussion by Adr0it** [IBMa/equal-access#1836] [Q&A]
Hello, I'm not the best programmer, so I may be making obvious mistake somewhere, but I'm trying to generate an excel report with your npm package. Unfortunately, no xlsx is being generated. The same goes for csv. The only thing that generates is the corresponding html and json. Here are pictures.
![image](https://github.com/IBMa/equal-access/assets/46908309/2923cc32-c506-4cf2-a63c-8387b37b9019)
![image](https://github.com/IBMa/equal-access/assets/46908309/83ced61d-4fea-4667-b42a-7e345264934b)
![image](https://github.com/IBMa/equal-access/assets/46908309/72fe6062-40ec-4bcd-ac2f-f3a96732dd9a)
![image](https://github.com/IBMa/equal-access/assets/46908309/92d4ed8f-d9d2-463d-8702-f260f5e0b352)

I'm trying to generate the Excel file that is generated by the Chrome extension, except programmatically.

**Comment by MHoov** [comment]
See #1837
